### PR TITLE
add comments on the review card without styling

### PR DESCRIPTION
**Pull request by MikeDobinson**
ticket 6

### CODE REVIEW

File: src/components/CommentList.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+export default function CommentList({ review_id }) {
+  const [comments, setComments] = useState([]);
+  useEffect(() => {
+    api.fetchCommentsByReviewId(review_id).then((comments) => {
+      setComments(comments);
+    });
+  }, [review_id]);
+
+  if (!comments.length)
```
Comment (roisinoreilly) [inline src/components/CommentList.jsx:13]: Nicely handled

File: src/components/CommentList.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    return (
+      <div>
+        <h2>Comments for review {review_id} </h2>
```
Comment (roisinoreilly) [inline src/components/CommentList.jsx]: This is super picky but I don't feel like the user needs to see the review ID, if we're cloning reddit essentially, each post would have an ID in the database but it's not shown to the user

**Review by roisinoreilly — APPROVED** [review]
Looks great! Good job handling if there's no comments on a review. Merge when ready :)

File: src/components/CommentList.jsx (hunk 2 above)
Comment (MikeDobinson) [reply src/components/CommentList.jsx]: Ah yeah, that was for my benefit making sure that it was working correctly - will remove it now!

**Pull request merged**